composer/src/main/kotlin/com/gojuno/composer/Args.kt

```diff
@@ -1,150 +1,118 @@
-package com.gojuno.janulator
+package com.gojuno.composer
 
+import com.beust.jcommander.IStringConverter
 import com.beust.jcommander.JCommander
 import com.beust.jcommander.Parameter
-import com.gojuno.composer.Exit
-import com.gojuno.composer.exit
 
 data class Args(
-        val appApkPath: String,
-        val testApkPath: String,
-        val testPackage: String,
-        val testRunner: String,
-        val shard: Boolean,
-        val outputDirectory: String,
-        val instrumentationArguments: List<Pair<String, String>>,
-        val verboseOutput: Boolean,
-        val devices: List<String>,
-        val devicePattern: String
+        @Parameter(
```

Comment:
btw I've started to use https://github.com/xenomachina/kotlin-argparser instead of JCommander, so far so good, you might want to take a look :)

cc @ming13

Reply:
Thanks, will check it